packages/block-library/src/container/index.js

```diff
@@ -0,0 +1,159 @@
+/**
+ * External dependencies
+ */
+import classnames from 'classnames';
+
+/**
+ * WordPress dependencies
+ */
+import { withFallbackStyles } from '@wordpress/components';
+import { compose } from '@wordpress/compose';
+import { __, sprintf } from '@wordpress/i18n';
+import {
+	getColorClassName,
+	withColors,
+	ContrastChecker,
+	InnerBlocks,
+	InspectorControls,
+	PanelColorSettings,
+} from '@wordpress/editor';
+import { Fragment } from '@wordpress/element';
+
+const applyFallbackStyles = withFallbackStyles( ( node, ownProps ) => {
+	const { textColor, backgroundColor } = ownProps.attributes;
+
+	return {
+		fallbackBackgroundColor: backgroundColor,
+		fallbackTextColor: textColor,
+	};
+} );
+
+export const name = 'core/container';
+
+export const settings = {
+	title: sprintf(
+		/* translators: Block title modifier */
+		__( '%1$s (%2$s)' ),
+		__( 'Container' ),
+		__( 'beta' )
```

Comment:
Is the UI freeze a block freeze? In which case, I don't think we should want anything "beta"-denoted.

Reply:
Good catch 👍